standardfiles/cookbook/.gitignore

```diff
@@ -37,15 +37,12 @@ doc/
 # chef stuff
 Berksfile.lock
 .kitchen
-.kitchen.local.yml
+kitchen.local.yml
 vendor/
 .coverage/
 .zero-knife.rb
 Policyfile.lock.json
-Cheffile.lock
-.librarian/
 
 # vagrant stuff
 .vagrant/
 .vagrant.d/
-.kitchen/
```

Comment:
Why take this out? It will commit peoples local kitchen converged etc?

Reply:
It's on line 39 too